spec/support/haiku_helper.rb

```diff
@@ -8,6 +8,16 @@ def submit_haiku(poem = 'A haiku', by: nil)
     fill_in 'Your haiku', with: poem
     click_on 'Send'
   end
+
+  def approve_haiku(poem, by: nil)
+    # TEMP this should eventually use admin approve view:
+    submissions = Submission.where(body: poem)
+    submissions = submissions.where(submitted_by: by) if by.present?
+
+    return unless submissions.one?
+
+    submissions.first.update!(status: Submission.statuses[:approved])
```

Comment:
`enum` should let you do…

```suggestion
submissions.first.approved!
```